packages/web-app/src/views/questionnaire-check-and-confirm.njk

```diff
@@ -0,0 +1,33 @@
+{% extends "layouts/main.njk" %}
+
+{% from "components/button-group.njk" import buttonGroup %}
+{% from "components/breadcrumb.njk" import breadcrumb %}
+{% from "components/confirmlist.njk" import confirmlist %}
+
+
+{% block pageTitle %}
+    Check and confirm - Appeal a householder planning decision - GOV.UK
+{% endblock %}
+
+{% block backButton %}
+  {{ breadcrumb(breadcrumbs) }}
+{% endblock %}
+
+{% block content %}
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <h1 class="govuk-heading-xl">
+                Review questionnaire
+            </h1>
+            <h2 class="govuk-heading-l">Check and confirm</h2>
+            {{ confirmlist(sections) }}
+        </div>
+    </div>
+    {% if reviewOutcome === 'COMPLETE' %}
+        <h3 class="govuk-heading-m">Confirming the questionnaire as complete makes the appeal available to inspectors </h3>
```

Comment:
It looks like this text should use the `govukWarningText` component rather than an `h3`

Reply:
The HTML can be used from the prototype but we should check it before using it in the app.

Looking at the prototype that text isn't there, although it is in the screenshot so probably need to check this with Anthony but assuming that we need it then it should use this component as it'll give the text the correct styling including the exclamation mark icon.

Reply:
True, I'll contact him